sw/airborne/subsystems/imu/imu_mpu9250_spi.c

```diff
@@ -194,6 +204,7 @@ void imu_mpu9250_event(void)
     imu_scale_mag(&imu);
     AbiSendMsgIMU_MAG_INT32(IMU_MPU9250_ID, now_ts, &imu.mag);
```

Comment:
Yes, You are right, I test all in V5.4.  but when i commit, I find the Master use AbiSendMsg instead of some Valid flag. 
BTW: should i pull Request again?